projects/icons/src/declarations/constants/icons/app-create-task-icon.const.ts

```diff
@@ -0,0 +1,9 @@
+import { IconDefinition } from '../../../declarations/interfaces/icon-definition.interface';
```

Comment:
а почему импорты разные? Иконки вроде лежат по соседсству и можно 

```suggestion
import { IconDefinition } from '../../interfaces/icon-definition.interface';';
```